internal/version/version.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"runtime"
+	"runtime/debug"
+)
+
+var (
+	// Name is the name of the binary. This can be overridden by the build
+	// process.
+	Name = "jvs"
+
+	// Version is the main package version. This can be overridden by the build
+	// process.
+	Version = "source"
```

Comment:
Hi, just wondering what are other Version values we are expecting other than "source", and when the Name and Version vars will be overridden?

Reply:
Hey @sqin2019 - they are overwritten by the build command: https://github.com/abcxyz/jvs/blob/3103e051920d7949d4b692b070032c72f43314de/.goreleaser.yaml#L38-L40

This allows us to inject the git tag and git commit directly into the build.

Reply:
Got it, thank you!